File: src/backend/playlist-server/src/main/java/com/sgwannabe/playlistserver/external/feign/dto/AlbumDTO.java
package com.sgwannabe.playlistserver.external.feign.dto;

import java.time.LocalDateTime;
import lombok.AccessLevel;
import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
public class AlbumDTO {
    private final Long id;
    private final AlbumType type;
    private final String title;
    private final String coverUrl;
    private final LocalDateTime releasedAt;

    private final ArtistDTO artist;
}

File: src/backend/streaming-server/src/main/kotlin/com/lalala/streaming/dto/AlbumDTO.kt
package com.lalala.streaming.dto

import java.time.LocalDateTime

data class AlbumDTO (
    val id: Long,
    val type: AlbumType,
    val title: String,
    val coverUrl: String,
    val releasedAt: LocalDateTime,
    val artist: ArtistDTO,
) {}
